package.json

```diff
@@ -18,21 +18,25 @@
     ]
   },
   "scripts": {
-    "build": "babel src/ -d build/",
+    "build": "babel src/ -d build/ --extensions \".js,.ts\" --source-maps inline",
+    "build:types": "tsc --emitDeclarationOnly",
     "prepublish": "yarn run build",
     "test": "jest",
     "flow": "flow",
-    "lint": "eslint '?(__mocks__|src|tests)/**/*.js'",
+    "lint": "eslint \"?(__mocks__|src|tests)/**/*.+(ts|js)\"",
     "ci": "yarn flow && yarn lint && yarn test --coverage",
```

Comment:
Does `yarn type-check` currently fail? Otherwise I would add it to `ci`.

Reply:
I'll check. It might because I added the allowJs flag in tsconfig.json. But if it works without that flag I'll definitely add it.